### ISSUE TITLE
Option to ignore bait listings in search results, crafting cost and price history

### ISSUE THREAD

**Issue by hollo6** [Auctionator/Auctionator#1597]
### What version(s) of WoW are you using?

Retail

### What problem are you experiencing that led to you asking for this feature?

With the enormous amount and constant flow of bait listings, it is almost always impossible to have a proper search result when looking at multiple items, e.g. shopping lists or craft search (from crafting window Search button). Consequences:
- Crafting cost calculations are way off from actual achievable sums.
- Shopping lists only show the baits, so actual achievable prices can only be determined by manually checking every tier/quality and remembering their individual results from their own pages.
- Price history is mostly useless.

Redoing the search is not a solution either because even if one of the baits are gone on the second/third/nth search, some others will have them by then for sure.

### What solution would you like?

Best solution for me would be a configurable threshold below which the results would be ignored (not shown in shopping list results and not used for crafting cost calculations), e.g. "ignore the price of these listings in calculations if there are < X items listed at a >Y% lower price than the second lowest result".

### Any alternatives you can think of?

I'm sure there are more clever and/or robust ways to achieve this, but even something very basic like this would make humans' life a lot less miserable.

**Comment by UnaffiliatedGamer** [comment]
I completely agree and came here looking for a solution to the same problem. These bait-listings are an insufferable nuisance.
Is it possible to easily identify and ignore these outliers? If not, even picking up the second lowest value instead of the lowest might be a start. Or possibly an option to manually override an items value in your database.

**Comment by hollo6** [comment]
For me, manual price setting would not help at all as it would need even more work than just checking every individual item.

**Comment by UnaffiliatedGamer** [comment]
Yeah manually editing would be a chore, it was a bad idea. It looks like we can kind of do it already as well. If you own some of each item and then bring up the auctionator page as if selling it, you can reselect that item to refresh its saved price. Repeat this if you have an abnormal price and move on to the next reagent.

**Comment by kukabay** [comment]
I came here to open this exact topic, but everyone is voicing the same problem. I hope the developer solves it, because it's actually very simple.

Because of the price cutters, it constantly reflects the lowest price and this causes inaccuracy in all rates on the sheet.

Another solution different from the one mentioned above is that it reflects the number we enter in the Quantity section and I think this will be more accurate. Because crafters always buy bulk products, the important thing is the cost of that product to us in the amount we want. For example, if I chose to buy 1000 products, it should reflect the price of those 1000 products to me. (a small box tick mark can be added next to where we write - integrate into the price style)

![t7](https://github.com/user-attachments/assets/29569b6b-b8d5-4fa2-a8bb-1877894fb35c)

**Comment by hollo6** [comment]
I actually do not bulk craft, just do everything in small quantitied for patron orders and myself, so if I need like 10 items for that craft and the baiters have 4 listed, it would still be a very skewed price - except if you mean to record the price that the full needed quantity could be bought for.
But even so, without a global value for this, it would still need a lot of manual work to set these for every tier of every individual item.

**Comment by kukabay** [comment]
Yes, that's why there should be 2 separate options for both individual and bulk purchasers. The solution you offer as an individual and the solution I offer as a bulk purchase is the most logical. It's actually simple to do, hopefully the developer can add it soon.

**Comment by hollo6** [comment]
> **Question, if these were the prices listed (GG.ss) for an Reagant on your realm what value would you want reflected as the price? 10 gold? Or is that volume too low, so 10.98?**
> 
> ```
>     {price = 6.26, volume = 1},
>     {price = 6.27, volume = 2},
>     {price = 10.00, volume = 28},

For this case, I want the 10. But in my proposal, there is a configurable quantity to exclude lowest prices below it; I would most probably set it to 5, maybe 10. Both would result in the 10 being the recorded value.
I believe this would also work for bulk crafters just as well, they would just have to set that quantity to 1000 or whatever.

> Maybe an option to only apply this kind of pricing logic to reagents would be necessary.

That would be perfectly fine, as bait listing is almost exclusively a reagents problem.

**Comment by plusmouse** [comment]
There isn't a good solution, as the Blizzard interface doesn't provide details of all the auctions for a given item without an additional scan, which would make the shopping search take much much longer.

**Comment by plusmouse** [comment]
The initial search results including multiple items only supply the lowest price

**Comment by hollo6** [comment]
Then how about giving us an option to make a "full" search if making a shopping list or crafting search?

**Comment by kukabay** [comment]
> Then how about giving us an option to make a "full" search if making a shopping list or crafting search? I could live with it taking as long as it takes if it meant getting accurate results.

I agree with that. There is nothing negative about it taking longer. It can even take a minute or two, as long as it's accurate.

I have 6 professions. I have a googlesheet table for each profession and each one has a list of 10-15 materials.
However, every time I use this feature, I have to check it one by one. Because it shows the 10g product as 6g as shown above and that's why all the googlesheet statistical information is wrong.

So even if it will take a long time, if you can do it, please do it ^^ this could be an additional option and those who want to can do it (which I am sure everyone will do)

**Comment by randyzie** [comment]
Would love a 'mean' price option
